autoload/jetpack.vim

```diff
@@ -36,7 +36,7 @@ let g:jetpack#copy_method =
   " symlink  : fs_symlink (nvim only)
   " hardlink : fs_link (nvim only)
 
-let s:packages = []
+let s:packages = {}
```

Comment:
The 'name' property is redundant. Please consider dropping it.
`for ... in values(...)` can be replaced by `for [key, value] in items(...)`.

Reply:
OK!